daffodil-test/src/test/resources/org/apache/daffodil/section02/validation_errors/Validation.tdml

```diff
@@ -407,8 +424,33 @@
         </xs:restriction>
       </xs:simpleType>
     </xs:element>
-    
-    <xs:element name="e3_2" dfdl:lengthKind="delimited">
+
+    <xs:element name="e2_6" dfdl:lengthKind="delimited" dfdl:encoding="iso-8859-1">
+        <xs:simpleType>
+            <xs:restriction base="xs:hexBinary">
+                <xs:minLength value="7"/>
+                <xs:maxLength value="7"/>
+            </xs:restriction>
+        </xs:simpleType>
+    </xs:element>
+
+    <xs:element name="e2_7" dfdl:lengthKind="delimited" dfdl:encoding="iso-8859-1">
+        <xs:simpleType>
+            <xs:restriction base="xs:hexBinary">
+                <xs:length value="7"/>
+            </xs:restriction>
+        </xs:simpleType>
+    </xs:element>
+
+    <xs:element name="e2_8" dfdl:lengthKind="delimited" dfdl:encoding="iso-8859-1">
+        <xs:simpleType>
+            <xs:restriction base="xs:hexBinary">
+                <xs:length value="0"/>
+            </xs:restriction>
+        </xs:simpleType>
+    </xs:element>
```

Comment:
Please note that e2_8 is a type derived from hexBinary like the others, but the comment on the test case that uses e2_8 says the type "int" which indicates either e2_8 should be a type derived from int or that comment is incorrect.  

Also please note that element names like e2_2, etc., are not well chosen or easily maintainable names.  You can't insert new elements between them easily without having to renumber or tack on new numbers.  In fact, the numbers tacked on the original element names seem like they were intended to provide some information about the facets, not just give some kind of ordering to the elements, see this example below:

```scala
    <xs:element name="e2_2" dfdl:lengthKind="delimited">
      <xs:simpleType>
        <xs:restriction base="ex:stMinLength_1">
          <xs:minLength value="2" />
        </xs:restriction>
      </xs:simpleType>
    </xs:element>
```

A name like `ex:stMinLength_1` is much more clear that it's defining a string with a minLength of 1.  So, these tests would become easier to maintain if they used similarly descriptive names such as `hex_length_0` instead of `e2_8`.